BUILD.bazel

```diff
@@ -68,3 +69,15 @@ filegroup(
     ],
     visibility = ["//visibility:public"],
 )
+
```

Comment:
I'm okay with merging this PR, but I don't understand this well enough to commit to maintaining it yet. So this might change or be removed in the future.

If that's okay with you, please add a TODO:

```
# TODO(#762): investigate generating stardoc documentation from these files.
# bzl_library targets in this repository may change in the future, so use with caution.
```